cmd/server.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"log/slog"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+
+	"github.com/metal-stack/go-dhcp-relay/config"
+	"github.com/metal-stack/go-dhcp-relay/server"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	interfaceFlag = &cli.StringFlag{
+		Name:    "interface",
+		Aliases: []string{"i"},
+		Value:   "",
+		Usage:   "listening interface",
+		EnvVars: []string{"INTERFACE"},
+	}
+	dhcpServersFlag = &cli.StringSliceFlag{
+		Name:    "dhcp-servers",
+		Aliases: []string{"s"},
+		Value:   cli.NewStringSlice(),
+		Usage:   "list of dhcp servers to forward requests to",
+		EnvVars: []string{"DHCP_SERVERS"},
+	}
+	maximumHopCountFlag = &cli.UintFlag{
+		Name:    "maximum-hop-count",
+		Aliases: []string{"c"},
+		Value:   config.DefaultMaximumHopCount,
+		Usage:   "packets whose hop count exceeds this value will be dropped",
+		EnvVars: []string{"MAXIMUM_HOP_COUNT"},
+	}
+)
+
+func rootCmd(cCtx *cli.Context) error {
+	iface := cCtx.String("interface")
```

Comment:
```suggestion
	iface := cCtx.String(interfaceFlag.Name)
```
And similar on others